cobalt/media/base/metrics_provider.cc

```diff
@@ -151,6 +151,27 @@ void MediaMetricsProvider::ReportActionLatencyUMA(
       UMA_HISTOGRAM_TIMES("Cobalt.Media.SbPlayer.Destroy.LatencyTiming",
                           action_duration);
       break;
+    case MediaAction::SBDRM_DESTROY_SYSTEM:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbDrm.Destroy.LatencyTiming", action_duration,
+          base::TimeDelta::FromMicroseconds(500),
```

Comment:
I believe this is quite low compared to the 3.77ms estimate you give in the PR description

Reply:
Yes, this is an extra margin for the lower bound.